classes-recipe/image_types_tegra.bbclass

```diff
@@ -32,7 +32,7 @@ def tegra_bootcontrol_overlay_list(d, bup=False, separator=','):
     overlays = d.getVar('TEGRA_BOOTCONTROL_OVERLAYS').split()
     if d.getVar('TEGRA_UEFI_USE_SIGNED_FILES') == "true":
         overlays.append('UefiDefaultSecurityKeys.dtbo')
-        if bup and os.path.exists('UefiUpdateSecurityKeys.dtbo'):
+        if bup and os.path.exists(os.path.join(d.getVar('STAGING_DATADIR') + '/tegra-uefi-keys', 'UefiUpdateSecurityKeys.dtbo')):
```

Comment:
Let `os.path.join` form the full path for you here with
```
os.path.join(d.getVar('STAGING_DATADIR'), 'tegra-uefi-keys', 'UefiUpdateSecurityKeys.dtbo')
```

Reply:
done